sycl/include/sycl/detail/type_traits.hpp

```diff
@@ -157,7 +156,7 @@ template <typename T, int N> struct get_elem_type_unqual<vec<T, N>> {
 template <typename VecT, typename OperationLeftT, typename OperationRightT,
           template <typename> class OperationCurrentT, int... Indexes>
 struct get_elem_type_unqual<SwizzleOp<VecT, OperationLeftT, OperationRightT,
-                               OperationCurrentT, Indexes...>> {
+                                      OperationCurrentT, Indexes...>> {
```

Comment:
Strictly speaking, there should be no unrelated formatting changes, but in this particular case it is more of a nitpicking and can be ignored.

Reply:
the formatting task failed until I called clang-format -i per changed file

strictly:
```bash
cd ../llvm &&
git diff --cached --name-only --diff-filter=ACM | grep '\.[ch]pp$'|xargs -I{} clang-format -i```

Reply:
Topic will be resolved with next commit.

The clang-format tool could be used for essential code as Alexei Sachkov has shared
 
Previously used way is patching only the changed files but not the file regions
```
export PATH="$(pwd)/build/bin:$PATH"
git diff -U0 --no-color --relative HEAD^ | $(find -name clang-format-diff.py) -p1 -i
```